Source/ExecutionEngine/ExecutionEngine.cs

```diff
@@ -1244,6 +1244,13 @@ private static void CleanupCheckers(string requestId)
             verificationResult.Errors = null;
             verificationResult.Outcome = VCGen.Outcome.Inconclusive;
           }
+          catch(AggregateException e)
+          {
+            printer.AdvisoryWriteLine("Advisory: {0} SKIPPED because: {1}",
+              impl.Name, e.Message);
+            verificationResult.Errors = null;
+            verificationResult.Outcome = VCGen.Outcome.Inconclusive;
```

Comment:
Could we set the default of verificationResult.Outcome to Inconclusive? Right now the default is Outcome.Correct

Reply:
Oh, yeah, definitely! My original fix did set it to something else (a new constructor) and I was somewhat dissatisfied with losing that. But I've just checked and the test suite still passes if it's initialized to `Inconclusive`. 🎉

Reply:
Ah, but the Dafny test suite completely fails with this change. I'll need to dig deeper to figure out what's going on there.

Reply:
And so does the Boogie test suite. I was just running the wrong one previously. The bottom line is that the following code doesn't actually change the outcome when a proof succeeds:

https://github.com/boogie-org/boogie/blob/6251cd8ae3a87866f31d705fe8c30ed57c053dab/Source/VCGeneration/Split.cs#L1306-L1309

I think that solving this correctly would require a substantially different approach to tracking the outcome (which I'd like to do, but I'm not yet sure the best way to do). There's no single value that could be assigned at that would be correct in this location.

My current best guess at a solution would be to change it so that Boogie constructs a list of (proof obligation, result) pairs, and builds the overall result out of a combination of those. That'll require some substantial work, though.

So, my inclination would be to merge this, along with your other two suggestions (a new outcome alternative and removed comment) but to leave that refactoring to a future PR (which I will do, since it'll be really nice to have for other future work).

Reply:
Sounds good, thanks :-)